common/lib/xmodule/xmodule/templates/problem/latex_problem.yaml

```diff
@@ -2,18 +2,25 @@
 metadata:
     display_name: Problem Written in LaTeX
     source_code: |
-        % Nearly any kind of edX problem can be authored using Latex as
-        % the source language.  Write latex as usual, including equations.  The
-        % key new feature is the \edXabox{} macro, which specifies an "Answer
-        % Box" that queries students for a response, and specifies what the
-        % epxected (correct) answer is.
+        % Nearly any kind of edX problem can be authored using LaTeX as
+        % the source language.  Write LaTeX as usual, including equations. The
+        % key new feature is the \edXabox{} macro, which specifies an "answer
+        % box" that queries students for a response, and specifies what the
+        % expected (correct) answer is.
 
```

Comment:
I don't actually see this text anywhere. If it is visible I'd suggest rephrasing so that we don't identify the feature as "new"